### PR TITLE
Update dependencies to support Rails 5

### PR DESCRIPTION
**Pull request by nykma**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 language: ruby
 
 rvm:
-  - 2.2
```
Comment (remiprev) [inline .travis.yml]: Is there any way we could keep `2.2.2`?

File: activerecord_json_validator.gemspec [hunk 2]
```diff
@@ -18,15 +18,15 @@ Gem::Specification.new do |spec|
...
 
-  spec.add_development_dependency 'bundler', '~> 1.3'
+  spec.add_development_dependency 'bundler', '~> 1.12'
   spec.add_development_dependency 'rake'
-  spec.add_development_dependency 'rspec', '~> 3.1'
+  spec.add_development_dependency 'rspec', '~> 3.5'
   spec.add_development_dependency 'pg'
   spec.add_development_dependency 'mysql2'
-  spec.add_development_dependency 'activesupport', '>= 4.1.0', '< 5'
+  spec.add_development_dependency 'activesupport', '>= 4.1.0', '< 6'
```
Comment (remiprev) [inline activerecord_json_validator.gemspec]: I think we could match `activesupport` version requirement with `activerecord`’s.

**Comment by remiprev** [comment]
I’m going to merge this and fix the small things I commented on. Thanks! 😄

**Comment by nykma** [comment]
Thank you! :smile:

**Pull request merged**